crates/pallet-dummy-precompiles-code/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+//! A substrate pallet.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::traits::StorageVersion;
+pub use pallet::*;
+use sp_core::H160;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
+/// The dummy code used to be stored for precompiles addresses - 0x5F5FFD (as raw bytes).
+///
+/// This is actually a hand-crafted sequence of opcodes for a bare-bones revert.
+/// The REVERT opcode (which is FD) - it takes two arguments from the stack with PUSH0 (5F twice).
+pub const DUMMY_CODE: [u8; 3] = [95, 95, 253];
```

Comment:
We should use the hex encoding here at the very least:

```suggestion
pub const DUMMY_CODE: [u8; 3] = [0x5F, 0x5F, 0xFD];
```

Maybe we could even use `hex_literal` macro here - would be even better.

Also, can't we do `[u8; _]` instead of `[u8; 3]`?

Reply:
https://github.com/humanode-network/humanode/pull/777/commits/7c8287454bada406a0c9902608a9746762ea19ad